code/modules/surgery/organs/tongue.dm

```diff
@@ -446,36 +448,46 @@
 	signer.verb_whisper = "subtly signs"
 	signer.verb_sing = "rythmically signs"
 	signer.verb_yell = "emphatically signs"
+	signer.bubble_icon = "signlang"
 	ADD_TRAIT(signer, TRAIT_SIGN_LANG, ORGAN_TRAIT)
 	REMOVE_TRAIT(signer, TRAIT_MUTE, ORGAN_TRAIT)
 
 /obj/item/organ/internal/tongue/tied/Remove(mob/living/carbon/speaker, special = 0)
 	..()
-	speaker.verb_ask = initial(verb_ask)
-	speaker.verb_exclaim = initial(verb_exclaim)
-	speaker.verb_whisper = initial(verb_whisper)
-	speaker.verb_sing = initial(verb_sing)
-	speaker.verb_yell = initial(verb_yell)
+	speaker.verb_ask = initial(speaker.verb_ask)
+	speaker.verb_exclaim = initial(speaker.verb_exclaim)
+	speaker.verb_whisper = initial(speaker.verb_whisper)
+	speaker.verb_sing = initial(speaker.verb_sing)
+	speaker.verb_yell = initial(speaker.verb_yell)
+	speaker.bubble_icon = initial(speaker.bubble_icon)
 	REMOVE_TRAIT(speaker, TRAIT_SIGN_LANG, ORGAN_TRAIT)
 
-//Thank you Jwapplephobia for helping me with the literal hellcode below
-
 /obj/item/organ/internal/tongue/tied/modify_speech(datum/source, list/speech_args)
+	// The message we send instead of our normal one
 	var/new_message
+	// The original message
 	var/message = speech_args[SPEECH_MESSAGE]
+	// Is there a !
 	var/exclamation_found = findtext(message, "!")
+	// Is there a ?
 	var/question_found = findtext(message, "?")
-	var/mob/living/carbon/signer = owner
 	new_message = message
 	if(exclamation_found)
 		new_message = replacetext(new_message, "!", ".")
 	if(question_found)
 		new_message = replacetext(new_message, "?", ".")
 	speech_args[SPEECH_MESSAGE] = new_message
 
-	if(exclamation_found && question_found)
-		signer.visible_message(span_notice("[signer] lowers one of [signer.p_their()] eyebrows, raising the other."))
+	if(question_found)
+		tonal_indicator = mutable_appearance('icons/mob/talk.dmi', "signlang1", TYPING_LAYER)
 	else if(exclamation_found)
-		signer.visible_message(span_notice("[signer] raises [signer.p_their()] eyebrows."))
-	else if(question_found)
-		signer.visible_message(span_notice("[signer] lowers [signer.p_their()] eyebrows."))
+		tonal_indicator = mutable_appearance('icons/mob/talk.dmi', "signlang2", TYPING_LAYER)
```

Comment:
Should I keep the visible message if I make it an indicator pref then? Since if I add that pref, then people who have it off won't be able to tell if someone's asking a question or not.

Reply:
done